wiring/inc/spark_wiring_ble.h

```diff
@@ -945,6 +945,8 @@ class BleLocalDevice {
     int setScanParameters(const BleScanParams& params) const;
     int getScanParameters(BleScanParams* params) const;
     int getScanParameters(BleScanParams& params) const;
+    int setScanCoded(bool use_coded) const;
```

Comment:
If we're going to support coded PHY for scanner, we should also support coded PHY for advertiser in Device OS.

Reply:
Agree this would be nice, but to keep changes manageable, I would prefer to get Coded PHY scanning fully working and accepted, then start a PR for Coded PHY advertising.

Reply:
I'm wondering what device you were using to act as the advertiser to verify the coded PHY scanning. If Particle devices can also support coded PHY for advertiser that would be nice and I can also help verify this PR.

Reply:
I used a custom sensor based on the same nRF52840 chip. You could also use a smartphone w/nRF Connect App but only some of the newest phones support BLE 5 Coded Phy. If you indicate that this PR is getting stable, I will work on a separate one to enable advertising in Coded Phy with the Boron.

Reply:
I'm afraid that I cannot use the nRF Connect App to simulate an advertiser to be advertising on the coded PHY. I'm looking forwards to seeing another PR that implements the coded PHY for advertiser in Device OS.

Reply:
My phone supports Coded PHY, and I was able to use the nRF Connect App to advertise using it. A Tracker using this code successfully scanned it. 

I think the advertiser in Device OS would also be ideal because it will allow us to expand the test cases we can do.

Reply:
@mgolu Thanks for confirming that! I have another Particle Boron arriving today, and now have code ready for **both** scanner and advertiser. As soon as I get things tested using these two Borons, I will update the code and await verification.